randovania/games/prime2/json_data/Temple Grounds.txt

```diff
@@ -1681,8 +1681,12 @@ Extra - in_dark_aether: False
       Trivial
   > Next to Pickup
       Any of the following:
-          Space Jump Boots or Standable Terrain (Beginner)
-          Morph Ball Bomb and Morph Ball and Bomb Jump (Beginner)
+          Space Jump Boots or Slope Jump (Beginner) or Standable Terrain (Beginner)
```

Comment:
I'll leave the concept of a NSJ slope jump in beginner for someone else, but I find it suspicious.

Reply:
You're entirely correct; someone else mentioned the slope jump being beginner, and I didn't actually test it before adding. I've upgraded it to Intermediate.

Reply:
i had mentioned it being intermediate not beginner.